oidc_provider/tests/cases/test_token_endpoint.py

```diff
@@ -839,6 +839,25 @@ def test_pkce_parameters(self):
 
         json.loads(response.content.decode('utf-8'))
```

Comment:
Is the original "good path" test here even asserting anything? I guess its checking for utf-8 encoded JSON, but nothing actually indicating a success.

I know it isn't strictly part of the change, but I feel like testing the behaviour without the `code_verifier` set is undermined a little by the other test not actually asserting a token is returned.